src/api/interactivity.js

```diff
@@ -83,15 +83,15 @@ export default class Interactivity {
     * To create a Interactivity object an array of {@link carto.Layer} is required.
     * Events fired from interactivity objects will refer to the features of these layers and only these layers.
     *
-    * @param {Array<carto.Layer>} layerList - Array of {@link carto.Layer}, events will be fired based on the features of these layers. The array cannot be empty, and all the layers must be attached to the same map.
+    * @param {carto.Layer|Array<carto.Layer>} layerList - {@link carto.Layer} or array of {@link carto.Layer}, events will be fired based on the features of these layers. The array cannot be empty, and all the layers must be attached to the same map.
     *
     * @example
     * const myLayer = new carto.Layer('layer0', mySource, myStyle);
```

Comment:
I think `layer` instad myLayer is consistent with our doc

Reply:
I thought the same :+1: